src/main/java/io/confluent/connect/jdbc/source/TimestampIncrementingTableQuerier.java

```diff
@@ -249,6 +258,22 @@ private boolean isIntegralPrimitiveType(Object incrementingColumnValue) {
            || incrementingColumnValue instanceof Byte;
   }
 
+  /**
+   * @param columnName column name
+   * @param columnAlias optional column alias
+   * @param quoteString string used to quote SQL identifiers
+   * @return the quoted column with a prefixed alias, if applicable
+   */
+  private String getQuotedColumn(String columnName, String columnAlias, String quoteString) {
+    String result = "";
+
+    if (!columnAlias.isEmpty()) {
+      result += JdbcUtils.quoteString(columnAlias, quoteString) + ".";
```

Comment:
Yeah, I could see that working. Just to be clear, the change would essentially enable users to prevent column name configuration values from being quoted if they have already quoted it themselves?